### DISCUSSION TITLE
What are some examples of good documentation in open source projects?

### DISCUSSION THREAD

**Discussion by danieltott** [Virtual-Coffee/virtualcoffee.io#380] [Open Source]
We were talking in the Hacktoberfest Co-Working Room, and @danieljanderson asked a question about learning docs pretty well. We started talking about what goes in to good documentation, and I thought it would be fun to get a list of repos with **really good docs**!

@meg-gutshall started us off with a couple: 
- https://github.com/rubyforgood/casa
- https://github.com/meg-gutshall/borrow-buddy

I think the docs on https://github.com/Virtual-Coffee/virtualcoffee.io are getting pretty good too personally 😄 

What other repos have great docs?

**Comment by nickytonline** [comment]
There are some mentioned in this DEV post of mine, [Projects with Great Documentation](https://dev.to/nickytonline/projects-with-great-documentation-352h).
